coordinator/persistence-cassandra-4.1/src/main/java/io/stargate/db/cassandra/impl/Cassandra41Persistence.java

```diff
@@ -462,7 +463,7 @@ private <T extends Result> CompletableFuture<T> executeRequestOnExecutor(
               // we convert, and runOnExecutor will detect it's already converted.
               PersistenceException pe =
                   Conversion.convertInternalException((Throwable) ((ErrorMessage) response).error);
-              pe.setTracingId(response.getTracingId());
```

Comment:
This came due to git merge, from either 3.11 or dse-6.8 persistence: method not `public` for 4.1 (same as 4.0).